Component/Pet/Pet.js

```diff
@@ -37,7 +74,14 @@ export default function Pet({ petData }) {
       <StyledPetData>
         <h2>Name: {petData.name}</h2>
         <h2>Type: {petData.type}</h2>
-        <h2>Characteristics: {...petData.characteristics}</h2>
+        <h2>
+          Characteristics:{" "}
+          {petData.characteristics.map((characteristic, index) =>
+            index < petData.characteristics.length - 1
+              ? characteristic + ", "
+              : characteristic
+          )}
+        </h2>
```

Comment:
As mentioned in my previous review, please use semantic HTML. 

Additionally, I suggest to use an `ul` for your characteristics for better semantics.

```suggestion
        <h2>
          Characteristics:{" "}
          <ul>
          {petData.characteristics.map((characteristic, index) => (
            <li key={index}>{characteristic}</li>
          ))}
        </ul>
```